scripts/pull_github_pr.sh

```diff
@@ -70,10 +78,15 @@ check_jenkins_job_status() {
     ORANGE='\033[0;33m'
     NC='\033[0m'
     echo -e "${ORANGE}\nWARNING:${NC} $jenkins_job is not stable"
-    exit 1
+    echo "exiting script since $lastCompletedBuild is unstable, you can force it by running ./script/pull_github_pr.sh <pr num> --force"
+    if [ -z "$FORCE" ]; then
```

Comment:
This means that the second parameter "dog" will do the same thing as "--force". Please check just for "--force".